src/packet/packet_ms.cpp

```diff
@@ -434,6 +434,27 @@ AOPacket *PacketMS::validateIcPacket(AOClient &client) const
                 client.sendServerMessage("First statement reached.");
             }
         }
+        if (l_args[4] == "=") {
+            auto l_statement = area->jumpToStatement(area->statement());
+            l_args = l_statement.first;
+            l_progress = l_statement.second;
+            client.m_pos = l_args[5];
+
+            client.sendServerMessageArea(client_name + " moved to the current statement.");
+        }
+        if (l_args[4].contains(QRegularExpression("(>)(\\d+)"))) {
+            QRegularExpressionMatch match = QRegularExpression("(>)(\\d+)").match(l_args[4]);
```

Comment:
running the same regular expression twice. This is also a duplicate of the regular expression contained in `isTestimonyJumpCommand`; if the intent is to alter that functionality please do it there